var/spack/repos/builtin/packages/flann/package.py

```diff
@@ -0,0 +1,143 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class Flann(CMakePackage):
+    """FLANN is a library for performing fast approximate nearest neighbor
+    searches in high dimensional spaces. It contains a collection of
+    algorithms we found to work best for nearest neighbor search and a system
+    for automatically choosing the best algorithm and optimum parameters
+    depending on the dataset.
+
+    FLANN is written in C++ and contains bindings for the following languages:
+    C, MATLAB and Python.
+    """
+
+    homepage = "http://www.cs.ubc.ca/research/flann/"
+    url      = "https://github.com/mariusmuja/flann/archive/1.9.1.tar.gz"
+
+    version('1.9.1', '73adef1c7bf8e8b978987e7860926ea6')
+    version('1.8.5', '02a81640b1e9c11796a0413976dc11f5')
+    version('1.8.4', '774b74580e3cbc5b0d45c6ec345a64ae')
+    version('1.8.1', '1f51500e172f5e11fbda05f033858eb6')
+    version('1.8.0', '473150f592c2997e32d5ce31fd3c19a2')
+
+    # Options available in the CMakeLists.txt
+    # Language bindings
+    variant("python",   default=False,
+            description="Build the Python bindings. "
+                        "Module: pyflann. "
+                        "Python2 verified. "
+                        "Python3 claimed, not attainable.")
```

Comment:
You can remove these extra comments now that we got Python 3 working. `Build the Python bindings` is enough.